scripts/release.ts

```diff
@@ -37,6 +37,19 @@ const inquireReleaseType = async (): Promise<ReleaseType> => {
   return (await inquirer.prompt(questions)).releaseType
 }
 
+const inquireCleanBranchOffMain = async (): Promise<boolean> => {
```

Comment:
Should probably do more checks but conventions over enforcing will do for now, will take a look at automating this however before opening it